bot.js

```diff
@@ -9,7 +9,7 @@ if (process.env.TEST) {
   console.log(`[${process.env.USER} => ${config.name}]`);
 } else {
   const IrcClient = require('./interfaces/irc');
-  var client = new IrcClient(config.server, config.name, config.channels);
+  client = new IrcClient(config.server, config.name, config.channels);
```

Comment:
I added this for the sake of symmetry. But if this works, then it must be okay. What do you say? Which one is better for us?